### PR TITLE
set read only change

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by xiangyan99** [comment]
#8416

File: sdk/appconfiguration/azure-appconfiguration/HISTORY.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 
 -------------------
 
+## 2019-12-xx Version 1.0.0b6
+
+### Breaking changes
+
+- Combine set_read_only & clear_read_only to be set_read_only(Ture/False)
```
Comment (richardpark-msft) [inline sdk/appconfiguration/azure-appconfiguration/HISTORY.md]: Minor spelling mistake (Ture -> True)

Comment (xiangyan99) [reply sdk/appconfiguration/azure-appconfiguration/HISTORY.md]: Good catch!

**Comment by bryevdv** [comment]
My only comment, not knowing much about the guarantees of the service, is that tests that exercise calling `set_read_only` multiple times in a row with the same value might be nice

**Pull request merged**